controllers/componentdetectionquery_controller_conditions.go

```diff
@@ -31,59 +31,69 @@ import (
 
 func (r *ComponentDetectionQueryReconciler) SetDetectingConditionAndUpdateCR(ctx context.Context, req ctrl.Request, componentDetectionQuery *appstudiov1alpha1.ComponentDetectionQuery) {
 	log := ctrl.LoggerFrom(ctx)
+	var currentCDQ appstudiov1alpha1.ComponentDetectionQuery
+	err := r.Get(ctx, req.NamespacedName, &currentCDQ)
```

Comment:
Since we’re the only resource reconciling CDQs, we shouldn’t need to get the resource before updating it

Reply:
updated